src/Domain/LeanCode.DomainModels.MassTransitRelay/IPipeContextServiceResolver.cs

```diff
@@ -15,14 +15,14 @@ TService GetService<TService>(PipeContext pipe)
 
     public class AutofacPipeContextServiceResolver : IPipeContextServiceResolver
```

Comment:
Do we need this class anymore? It was used when we had to use service locator pattern in filters, but now we rely on MT.